pkg/fleet/installer/service/embedded/dd-cleanup

```diff
@@ -0,0 +1 @@
+exit 0
```

Comment:
missing shebang + comment that it's owned by datadog-apm-inject for users passing by